src/web/mjs/connectors/futabanet.mjs

```diff
@@ -0,0 +1,45 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class futabanet extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'futabanet';
+        super.label = 'がうがうモンスター (Futabanet Monster)';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://futabanet.jp';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'head title');
```

Comment:
Use `fetchDOM(request, 'h2.detail-ex__title')` instead, this will avoid the splitting